internal/compiler/builtins.slint

```diff
@@ -102,6 +102,9 @@ export component Text inherits Empty {
     in property <TextOverflow> overflow;
     in property <TextWrap> wrap;
     in property <length> letter-spacing;
+    in property <brush> stroke;
```

Comment:
As a follow-up note to self: It makes sense to split `Text` then into `Text` and `TextWithStroke` (or so), so that not every simple `Text` pays the price of the stroke properties.

Reply:
In all three implementations, I’m checking `stroke_brush.is_transparent()` before doing anything related to strokes, so there shouldn’t be any impact on `Text`s without an outline.

Reply:
Right, performance wise this is perfectly fine. I was thinking in terms of memory usage through additional Property<> fields that aren't used for all the cases where `Text` is just used as `Text { text: "Ok"; }`.

Either way, I also don't think this needs to be changed in this PR.